src/org/loklak/api/search/GenericScraper.java

```diff
@@ -48,6 +48,8 @@
 
 public class GenericScraper extends HttpServlet {
 
+	private static final long serialVersionUID = 4653635987712691127L;
```

Comment:
@sudheesh001 @shivenmian If this particular file patch is creating a problem. I can fix it right now.